components/connection_list/stylesheets/index.styl

```diff
@@ -0,0 +1,56 @@
+borderColor(color)
+  alpha(color, 30%)
```

Comment:
let's just codify the concept of the border color and extract this to a higher level file